src/web/services/userAccount.ts

```diff
@@ -80,6 +87,22 @@ export async function GetAllUsersOfParticipant(participantId?: number) {
   }
 }
 
+export async function RemoveUser(id: number) {
+  try {
+    await axios.delete(`/users/${id}`);
+  } catch (e: unknown) {
+    throw backendError(e, 'Could not delete user');
+  }
+}
+
+export async function UpdateUser(id: number, formData: InviteTeamMemberForm) {
+  try {
+    await axios.patch(`/users/${id}`, formData);
```

Comment:
As discussed - this is sending too much data, limit what's being sent to just the fields that can be updated.